worker/raft/raftforwarder/manifold.go

```diff
@@ -31,22 +90,82 @@ func (config ManifoldConfig) start(context dependency.Context) (worker.Worker, e
 	if err := context.Get(config.CentralHubName, &hub); err != nil {
 		return nil, errors.Trace(err)
 	}
-	return config.NewWorker(Config{
+
+	var stTracker workerstate.StateTracker
+	if err := context.Get(config.StateName, &stTracker); err != nil {
+		return nil, errors.Trace(err)
+	}
+	statePool, err := stTracker.Use()
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+
+	st := statePool.SystemState()
+
+	logPath := filepath.Join(agent.CurrentConfig().LogDir(), "lease.log")
+	if err := primeLogFile(logPath); err != nil {
+		// This isn't a fatal error, so log and continue if priming
+		// fails.
+		config.Logger.Errorf(
```

Comment:
If it's not fatal it should be a warning.

Reply:
Ok.